net/pdnsd/files/etc/config/pdnsd.conf

```diff
@@ -0,0 +1,24 @@
+global {
```

Comment:
`/etc/config` is supposed to only contain uci config files.  This one is pdnsd specific

Reply:
move to /etc done.

Reply:
If you are going to preserve this file, it should be at `files/pdnsd.conf`.